warrior/Actions/CommonActions/common_actions.py

```diff
@@ -198,7 +169,20 @@ def verify_key_already_exists_and_update(orig_dict, new_dict):
                 filepath = getAbsPath(filepath, os.path.dirname(testcasefile_path))
                 with open(filepath, "r") as json_handle:
                     json_doc = json.load(json_handle)
-                    if jsonkey in json_doc:
+                    #if bool_store_all is set to True, all content of given json file will be
```

Comment:
Please move code block from line 172 to 209 out of the ```with``` block.

Reply:
done